frontend/src/hooks/useLayoutManager.tsx

```diff
@@ -1,12 +1,7 @@
 import { useRef } from 'react';
-import { Box, Element } from 'opentok-layout-js';
 import LayoutManager from '../utils/layoutManager';
 
-export type GetLayout = (
-  containerDimensions: { height: number; width: number },
-  boxes: Element[]
-) => Box[];
-
+export type GetLayout = InstanceType<typeof LayoutManager>['getLayout'];
```

Comment:
This defines the type as the one used by the class method rather than redefining it here, since changes in the class were not picked up but this type checking. Now any changes in the class will show up where this is used.